pkg/constants/time.go

```diff
@@ -0,0 +1,97 @@
+package constants
+
+import "time"
+
+var (
+	DefaultTimeFormat = "2006-01-02 03:04 PM"
+	// TimeFormatStrings defines the format strings we will use for time.Parse
+	// Note we have excluded some from `time` that are 1) intended for network timestamps, which we do not expect users
+	// to want, or 2) those that contain time zone abbreviations; go currently disregards them, even with time/tzdata.
+	// Anything other than the "time.Local" location (e.g. "EST") ends up assumed to be UTC, it's a bit aggravating.
+	TimeFormatStrings = []string{
+		time.ANSIC,    // "Mon Jan _2 15:04:05 2006"
+		time.RubyDate, // "Mon Jan 02 15:04:05 -0700 2006"
+		time.RFC822Z,  // "02 Jan 06 15:04 -0700" // RFC822 with numeric zone
+		time.RFC1123Z, // "Mon, 02 Jan 2006 15:04:05 -0700" // RFC1123 with numeric zone
+		time.RFC3339,  // "2006-01-02T15:04:05Z07:00"
+		time.Kitchen,  // "3:04PM"
+		time.Stamp,    // "Jan _2 15:04:05"
```

Comment:
For DateOnly, I'm not setting 05:00:00, but UTC midnight is five hours ahead of your local midnight. :-)
I think I've got the others fixed. I had to switch regex libraries to one that supports lookahead, because they didn't parse as time formats, and the regex was then matching "14:04:00 MST" as "140400 minutes". 😬